### PR TITLE
Impl 1048

### LINKED ISSUE #1048: Reduce the space of new artical
screenshot_15
Please add condition to hide the P tag if no content. 

`<p>
		            		<?php echo get_the_content(); ?>
									</p>
									<p>
			            	<?php echo_downloaded_documents(); ?>
									</p>
									</br>
									<p>									
			            	<?php odm_echo_extras(); ?>
									</p>
`

### PR DESCRIPTION
**Pull request by Huyeng**
Fixes #1048

### CODE REVIEW

File: single.php [hunk 1]
```diff
@@ -32,17 +32,19 @@
...
+
+									if(get_the_content()):
+										echo "<p>".get_the_content()."</p>";
+									endif;
+
+									if(odm_echo_extras()):
+										odm_echo_extras();
+									endif;
+
+									if(echo_downloaded_documents()):
```
Comment (acorbi) [inline single.php]: @Huyeng the `echo_downloaded_documents` and `odm_echo_extras` functions do not return any value. Why are you then checking if it returns true? 

I believe you should just call it.

Comment (acorbi) [inline single.php]: Doesn't it actually output the content twice?

Comment (Huyeng) [inline single.php]: to fix is error: https://opendevelopmentcambodia.net/news/soldiers-police-in-mondolkiri-under-investigation-for-wood-smuggling/

if no content, the tag <p> should not be added.

Comment (Huyeng) [inline single.php]: Calling the echo_downloaded_documents() function, it will return the p tag event though no content.

Comment (acorbi) [inline single.php]: I understand the problematic but the implementation is not so clean. I am going to merge this for the moment and open a PR with a change suggestion for you to review a bit later today or during the weekend.

Comment (Huyeng) [inline single.php]: I think we can add the some condition in its function to prevent the displaying of p tag.

**Pull request merged**